tests/unit/test_cli.py

```diff
@@ -112,7 +112,9 @@ def test_initialize(monkeypatch: pytest.MonkeyPatch, cli_runner: CliRunner, flag
     if not flags:
         mock_builder_init_func.assert_called_with()
     else:
-        mock_openstack_init_func.assert_called_with(arch=config.Arch.X64, cloud_name="hello")
+        mock_openstack_init_func.assert_called_with(
+            arch=config.Arch.X64, cloud_name="hello", prefix=""
```

Comment:
For the test, would it be better if the prefix is set, as to test the prefix arguments is working?

Reply:
This is a unit test that does not make use of the app-name so I think this test having the prefix does not carry meaning for now :) The integration test on the image-builder-operator side should test this though :D Thank you for the review!